packages/arb-token-bridge-ui/src/state/app/state.ts

```diff
@@ -76,6 +75,12 @@ export interface MergedTransaction {
   }
 }
 
+export interface TeleporterMergedTransaction extends MergedTransaction {
+  l1ToL2MsgData?: ParentToChildMessageData
```

Comment:
In `TeleporterMergedTransaction`, `l1ToL2MsgData` and `l2ToL3MsgData` should not be optional. At the very least, `l1ToL2MsgData` shouldn't be.